### PR TITLE
Show number of documents and date of last update on genes page

### LINKED ISSUE #3234: On genes page, report nr of genes (by build) and date of last update
Might be interesting things to show to the users.

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3234

Before:

image

After: 

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/genes/views.py [hunk 1]
```diff
@@ -14,16 +14,8 @@
 def genes():
     """Render seach box for genes."""
     query = request.args.get("query", "")
-    hgnc_id = None
```
Comment (northwestwitch) [inline scout/server/blueprints/genes/views.py:17]: This code was moved to the controllers

**Comment by northwestwitch** [comment]
Reminder: show number of genes in build 37 and in build 38 as separate numbers!

**Comment by dnil** [comment]
Yeah, this would potentially have been very useful today!

**Comment by northwestwitch** [comment]
Let's use it!

Fixed count by build:

image

File: scout/server/blueprints/genes/templates/genes/genes.html [hunk 2]
```diff
@@ -15,7 +15,9 @@
 
 <div class="container-float">
     <div class="card panel-default w-100 mt-3 justify-content-center">
-        <div class="panel-heading">Genes</div>
+        <div class="panel-heading">Genes
+          <span style="float:right;">Number of documents: build37->{{nr_genes[0]}} | build38->{{nr_genes[1]}} - Last updated: {{last_updated.strftime('%d-%m-%Y %H:%M:%S')}}</span>
```
Comment (dnil) [inline scout/server/blueprints/genes/templates/genes/genes.html:19]: Good stuff! How about some font styles on this line instead/in addition to the punctuation, or at least avoid the `->`. "Number of documents" is probably a little more under-the-hood than helps the user. 😊 
 
```suggestion
          <span style="float:right;"> {{nr_genes[0]}} GRCh37 genes | {{nr_genes[1]}} GRCh38 genes | Last updated  {{last_updated.strftime('%d-%m-%Y %H:%M:%S')}}</span>
```

**Review by dnil — APPROVED** [review]
Looks good! A similar on diseases would feel symmetric, but that can be another PR.

**Comment by northwestwitch** [comment]
> Looks good! A similar on diseases would feel symmetric, but that can be another PR.

Unfortunately can't be done. Diseases and OMIM terms are not saved with an _id that is an MongoDB ObjectID, so we need to fix it in another way.

**Comment by northwestwitch** [comment]
UU cool! Was it you @henrikstranneheim ?

File: scout/server/blueprints/genes/templates/genes/genes.html (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/genes/templates/genes/genes.html:19]: YES, much better!

**Pull request merged**